README.md

```diff
@@ -201,7 +200,35 @@ To dig deeper into Flyte, refer to the [Documentation](https://docs.flyte.org/en
+|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Communication Channels**                                                                                                                                                                                                                                                               |
+| [Slack Org](https://forms.gle/UVuek9WfBoweiqcJA)                                                                                                                                                                                                                                         |
+| [Email list](https://groups.google.com/a/flyte.org/g/users)                                                                                                                                                                                                                              |
+| **Biweekly Community Sync**                                                                                                                                                                                                                                                              |
+| Flyte OSS Community Sync, every other Tuesday, 9am-10am PDT                                                                                                                                                                                                                              |
+| [Events calendar & Sign Up](https://calendar.google.com/calendar/embed?src=admin%40flyte.org&ctz=America%2FLos_Angeles)                                                                                                                                                                  |
+| [Zoom link](https://us04web.zoom.us/j/71298741279?pwd=TDR1RUppQmxGaDRFdzBOa2lHN1dsZz09)                                                                                                                                                                                                  |
+| Meeting notes and topics are captured in this [document](https://docs.google.com/document/d/1Jb6eOPOzvTaHjtPEVy7OR2O5qK1MhEs3vv56DX2dacM/edit#heading=h.c5ha25xc546e)                                                                                                                    |
+| Previous meeting [video recordings](https://www.youtube.com/channel/UCNduEoLOToNo3nFVly-vUTQ)                                                                                                                                                                                            |
```

Comment:
do you want to make this into 2 columns, the single column looks a little off?

Reply:
Sure. I will give it a try.